tips/21-08-2017/Readme.md

```diff
@@ -1,4 +1,4 @@
-We have [discussed][1] earlier how to significally reduce a bundle size which imports `lodash` inside. Now we will learn more (in 2 parts) about specific webpack options and settings used in order to reduce the bundle still not breaking the functionality too much.
+We have [discussed][1] earlier about how to significantly reduce the size of a bundle that imports `lodash` inside itself. Now we will learn more (in 2 parts) about specific webpack options and settings used in order to reduce the bundle still not breaking the functionality too much.
```

Comment:
Internet says me this is not correct:
https://www.englishforums.com/English/DiscussVsDiscussAbout/zlpwz/post.htm
https://english.stackexchange.com/questions/146833/is-discuss-about-grammatically-incorrect